content/id/docs/concepts/overview/object-management-kubectl/imperative-command.md

```diff
@@ -0,0 +1,132 @@
+---
+title: Pengelolaan Objek Kubernetes dengan Perintah Imperatif
+content_template: templates/concept
+weight: 20
+---
+
+{{% capture overview %}}
+Objek-objek Kubernetes bisa dibuat, diperbarui, dan dihapus secara langsung dengan menggunakan perintah-perintah imperatif yang ada pada *command-line* `kubectl`. Dokumen ini menjelaskan cara perintah-perintah tersebut diorganisir dan cara menggunakan perintah-perintah tersebut untuk mengelola objek *live*.
+{{% /capture %}}
+
+{{% capture body %}}
+
+## Kelebihan dan kekurangan
+
+Perintah `kubectl` mendukung tiga cara pengelolaan objek:
+
+* Perintah imperatif
+* Konfigurasi objek imperatif
+* Konfigurasi objek deklaratif
+
+Lihat [Pengelolaan Objek Kubernetes](/id/docs/concepts/overview/object-management-kubectl/overview/) untuk menyimak diskusi mengenai kelebihan dan kekurangan dari tiap cara pengelolaan objek.
+
+## Cara membuat objek
+
+*Tool* `kubectl` mendukung perintah-perintah berbentuk kata kerja untuk membuat beberapa tipe objek yang paling umum. Perintah-perintah tersebut diberi nama yang mudah dikenali oleh pengguna yang belum familiar dengan tipe-tipe objek Kubernetes.
```

Comment:
So we will standardize `tool` usage here?

Reply:
I think `perangkat` is generally understandable

Reply:
Sorry guys I just finally get back on this. Ya, I think it's better to use `perangkat` to translate `tools` from now on.